src/templates/BillOfLading/sample.ts

```diff
@@ -12,6 +12,7 @@ export const BillOfLadingSample: BillOfLading = {
         "https://action.openattestation.com/?q=%7B%22type%22:%22DOCUMENT%22,%22payload%22:%20%7B%22uri%22:%20%22https://api-ropsten.tradetrust.io/storage/33737b71-96f6-4019-b6c1-5fcea04fcc2a%22,%22key%22:%20%22edcbadf17301fe5dfe174496c7edf29e0702e2775ee919bbccb74d0dfd5a1b4b%22,%22permittedActions%22:%20%5B%22STORE%22%5D,%22redirect%22:%20%22https://dev.tradetrust.io%22%7D%7D"
     }
   },
+  scac: "MAEU",
```

Comment:
Ehhh MAEU -> SCAC right, this is still referencing Maersk when u use MAEU

https://www.bic-code.org/container-bic-code/maeu/

Reply:
Ohh okay, I will change the sample data scac to "SGPU". So in the config file for document creator, the default code will be SGPU.

Reply:
Done